Dockerfile

```diff
@@ -1,8 +1,36 @@
-FROM alpine:3.18
+# Note:
+# * YAMLScript doesn't yet run on alpine
+# * ubuntu:24.04 is 78.1MB
+# * debian:bookworm is 117MB
+#
+# This image will also be used for the YAMLScript track repo's GHA workflows.
 
-# install packages required to run the tests
-RUN apk add --no-cache jq coreutils
+FROM ubuntu:24.04
+
+# Install packages required to run the tests:
+RUN apt-get update \
+ && apt-get install -y apt-utils \
+ && apt-get install -y \
+        curl \
+        jq \
+        make \
+        perl \
+        wget \
```

Comment:
Why insall both curl _and_ wget? Are both needed?

Reply:
Fixed in 997087b